addon/mixins/table-header.js

```diff
@@ -105,6 +107,19 @@ export default Mixin.create({
     return this.getProperties(['iconAscending', 'iconDescending']);
   }).readOnly(),
 
+  init() {
+    this._super(...arguments);
+
+    let fixed = this.get('fixed');
+    let height = this.get('sharedOptions.height');
+
+    warn(
```

Comment:
Just checked `Ember.warn` in our production deployment. It's indeed an empty function. :smile: